nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -1,26 +1,43 @@
+from distutils.util import strtobool
+import os
+import sys
+
+from nautobot.core.settings import *  # noqa F401,F403
+
+
+def is_truthy(arg):
+    """Convert "truthy" strings into Booleans.
+    Examples:
+        >>> is_truthy('yes')
+        True
+    Args:
+        arg (str): Truthy string (True values are y, yes, t, true, on and 1; false values are n, no,
+        f, false, off and 0. Raises ValueError if val is anything else.
+    """
+    if isinstance(arg, bool):
+        return arg
+    return bool(strtobool(str(arg)))
```

Comment:
If we're going to bake this into the sample config, I'd prefer that this get moved into a util function. The trick is it needs to be imported from somewhere that won't result in an import loop.

Reply:
I forgot that we actually do have it in `nautobot.utilities.utils.is_truthy`. We should see if it can be safely imported into settings from there without introducing any circular dependencies?